public_html/wp-content/themes/pattern-directory/src/components/pattern-grid/card-skeleton.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Skeleton, SkeletonWrapper } from '../skeleton';
+import IconHeartFilled from '../icons/heart-filled';
+import { getFrameHeight } from '../pattern-thumbnail/canvas';
+
+const CardSkeleton = () => {
+	const [ height, setHeight ] = useState();
+	const wrapper = useRef();
+
+	useEffect( () => {
+		if ( wrapper.current ) {
+			setHeight( `${ getFrameHeight( wrapper.current ) }px` );
+		}
+	}, [ wrapper ] );
+
+	return (
+		<div ref={ wrapper }>
+			<SkeletonWrapper>
+				<Skeleton height={ height } />
+				<Skeleton height="14px" marginTop="1rem" />
+
+				<SkeletonWrapper styles={ { flexDirection: 'row', marginTop: '0.5rem', alignItems: 'center' } }>
```

Comment:
Why `styles` instead of `style`?

Reply:
I was thinking that since it was going to be passed on specifically, not like `<Component {...props} />`, that it maybe should be different, but thinking about it out loud, probably not necessary.

Reply:
Updated in a8e2ce7.